elephant/csd.py

```diff
@@ -1,3 +1,357 @@
 #!/usr/bin/env python
-'''Current source density estimation methods from extracellular potentials'''
-from .icsd.icsd import estimate_csd
+"""This script is used to generate Current Source Density Estimates
```

Comment:
Here it would be better to have a more thorough explanation of what CSD is. For example you could use the explanation you provided in the `README.md`. This is important since the documentation will appear on the autodocs (.__docs__) etc.